processor/otel/consumer.go

```diff
@@ -108,39 +100,54 @@ func (c *Consumer) convert(td consumerdata.TraceData) (metadata.Metadata, []tran
 		}
 		name := otelSpan.GetName().GetValue()
 		if root || otelSpan.Kind == tracepb.Span_SERVER {
-			transaction := model_transaction.Event{
+			tx := transaction.Event{
 				Id:        spanID,
 				ParentId:  parentID,
 				TraceId:   traceID,
 				Timestamp: startTime,
 				Duration:  duration,
 				Name:      &name,
+				Metadata:  md,
+			}
+			parseTransaction(otelSpan, hostname, &tx)
+			if tx.Timestamp.IsZero() {
+				tx.Timestamp = time.Now()
```

Comment:
It's a detail, but this sets the timestamp to a different time for every event within the same request. I'd prefer keeping the current logic of setting the same timestamp for all events within one request.

Reply:
ok. why that would be preferable?